src/showplanner/ImporterModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { FaTimes, FaFileImport } from "react-icons/fa";
+import Modal from "react-modal";
+import { Button } from "reactstrap";
+
+interface ImporterProps {
+  isOpen: boolean;
+  close: () => any;
+}
+
+// TODO: This needs updating to actually either provide the weighting channel values (less preferred)
+// or update the importer to work that out itself.
```

Comment:
Has this been sorted out yet?

Reply:
Not yet. Current aim is having Show Planner still available, but there's nothing in this PR stopping it from "just working" when that side is fixed (high on my priority list).